README.md

```diff
@@ -1,67 +1,5 @@
-Heroku Stack Images
-=========
+## Heroku Cedar Stack images
 
-The provided `bin/cedar.sh` is the basis of a cedar stack image.
+This repo holds recipes for building the [Heroku Cedar Stack](https://devcenter.heroku.com/articles/cedar) images. The recipes are also rendered into Docker images that are [available on Docker Hub](https://registry.hub.docker.com/u/heroku/cedar/). See Heroku Dev Center for up-to-date [lists of packages available on the Cedar stacks](https://devcenter.heroku.com/articles/cedar-ubuntu-packages).
```

Comment:
I'm not sure what the package list link intends. The source is always going to be more accurate and the versions will be different if you pull this yourself or from the Docker Hub.